service/handlers.go

```diff
@@ -0,0 +1,139 @@
+package service
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/bar41234/bar_book_service/datastore"
+	"github.com/bar41234/bar_book_service/models"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+const (
+	errorMsgIdNotFound         = "Error: Id was not found!"
+	errorMsgInvalidPutRequest  = "Error: Invalid PUT request"
+	errorMsgInvalidPostRequest = "Error: Invalid POST request"
+)
+
+func Ping(c *gin.Context) {
+	c.JSON(http.StatusOK, map[string]string{
+		"message": "Ping Pong!",
+	})
+}
+
+func GetBook(c *gin.Context) {
+	bookStore, _ := datastore.BooksStoreFactory()
+	id := c.Param("id")
+	if id == "" {
+		c.JSON(http.StatusNotFound, errorMsgIdNotFound)
+		return
+	}
```

Comment:
Part of the URL signature is the `id` - so it can not be that this parameter will not send - and therefore this test is unnecessary in some routes